src/components/BrandConfigProvider/styles/StylesProvider.tsx

```diff
@@ -23,7 +21,7 @@ export const useStyles = <T extends ReturnType<CreateStyles>>(
   const styles = merge(
     {},
     namedStyles,
-    (componentStyles as any)[namespace],
+    componentStyles[namespace],
```

Comment:
Restricting the names above makes us not have to cast because namespace is now `keyof ComponentStyles`